routes/project/location/postgres.rb

```diff
@@ -40,6 +40,33 @@ class Clover
         204
       end
 
+      r.patch do
+        authorize("Postgres:edit", pg.id)
```

Comment:
what do you think about disabling patching during a convergence is going on? It feels kinda dangerous/complex to allow changing the convergence target while trying to converge already, even though I cannot pinpoint an immediate problem at the moment.

Reply:
Convergence is long operation and I don't want to put users into a situation where they had to wait long time when they change their mind. It felt dangerous to me at the beginning as well but after thinking on it for a quite while, I feel confident that changing the target wouldn't cause any major issues.